### DISCUSSION TITLE
ban period or cooldown period

### DISCUSSION THREAD

**Discussion by walteroa2023** [qdm12/ddns-updater#530] [Q&A]
Hi all,
I'me newbie and just have my new domain transfered to dynV6.
When I run ddns-updater got a message " ban period or cooldown period", What does it mean?
Should I claim to dynV6 for this?

**Comment by beartom** [comment]
" ban period or cooldown period" maybe not the root cause. It just skip update for you in the period.
Try **"provider_ip":false**,  that works for dynV6 on my server.

**Comment by qdm12** [comment]
It means it sent too many requests to dynv6 and their servers likely blocked you temporarily. Check your logs for more details, perhaps with LOG_LEVEL=debug too.
A possible cause is the public ipv6 fetched is different on every period, because of the local adapter portion of the address (second half of ipv6 address); a recent solution to this is to use the json setting `"ipv6_suffix"` (see your provider document). And don't forget to pull the latest image, since this is a recent change.
